### PR TITLE
added pacbio longread file

### PR DESCRIPTION
**Pull request by tanyasarkjain**
want to add pacbio longread dna (with RG information) for testing with pbsv and other pacbio-based modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -507,6 +507,8 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
       - mini.5p--3p.bam: subsample of hifi reads from the public pacbio dataset [pbmc singlecell mini](https://downloads.pacbcloud.com/public/dataset/IsoSeq_sandbox/2022_pbmc_singlecell_mini/ccs.bam) where the primers where removed with LIMA.
       - test.sorted.bam: 35 reads from a [public PacBio Revio dataset of HG002](https://downloads.pacbcloud.com/public/revio/2022Q4/HG002-rep1/analysis/HG002.m84011_220902_175841_s1.GRCh38.bam) aligned to GRCh38.
       - test.sorted.bam.bai: BAM index for 'test.sorted.bam'
+      - sample_pacbio_longread_dna.aligned.bam: WGS aligned reads generated by pbmm2, containing RG information
```
Comment (jfy133) [inline README.md:510]: Whas is the source of the reads used to align, and what is the reference genome - is it the `genome.fasta` here: https://github.com/nf-core/test-datasets/tree/1991fd4939c86871d116eb74276b2f00b5388d65/data/genomics/homo_sapiens/genome

For example?

Comment (fellen31) [reply README.md:510]: You could also consider re-aligning `test.sorted.bam` to genome.fasta with pbmm2 and updating it (with RGs), if that file contains enough reads for you. The missing RGs in `test.sorted.bam` was an oversight by me.

**Comment by tanyasarkjain** [comment]
Thanks - test.sorted.data is aligned to the entire hg38.fa I believe and the 35 reads are spread out across the genome, so I opted instead to use publicly available puretagrget pacbio reads from https://downloads.pacbcloud.com/public/dataset/PureTargetRE/Coriell/PBMM2-BAM-Input-For-IGV-And-TRGT/ and downsample and created a new fasta genome3.fasta with the coordinates (chr19:45760000-45770300) of these  reads, as the module I'm working on also requires a fasta input

File: README.md (hunk 1 above)
Comment (tanyasarkjain) [reply README.md]: Hi - I changed the datasets so they were based off the publicly available https://www.pacb.com/connect/datasets/ PURETARGET REPEAT EXPANSION PANEL WITH CORIELL SAMPLES. 
The data is aligned to genome3.fasta which is the segment of hg38.fa that corresponds to (chr19:45760000-45770300. In addition to the test.sorted.bam not containing read groups, the 35 were and aligned to hg38.fa, and corresponded to many different chromosomes/locations, + I found adding RG info to this dataset was not super trivial, so I opted to create new sample data for the purpose of testing pbsv.

**Comment by fellen31** [comment]
> Thanks - test.sorted.data is aligned to the entire hg38.fa I believe and the 35 reads are spread out across the genome

Yes, sorry. You are right, I forgot I needed it to be aligned to "real" coordinates of GRCh38 to use it as input for Paraphase.

File: README.md [hunk 2]
```diff
@@ -507,6 +508,8 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
       - mini.5p--3p.bam: subsample of hifi reads from the public pacbio dataset [pbmc singlecell mini](https://downloads.pacbcloud.com/public/dataset/IsoSeq_sandbox/2022_pbmc_singlecell_mini/ccs.bam) where the primers where removed with LIMA.
       - test.sorted.bam: 35 reads from a [public PacBio Revio dataset of HG002](https://downloads.pacbcloud.com/public/revio/2022Q4/HG002-rep1/analysis/HG002.m84011_220902_175841_s1.GRCh38.bam) aligned to GRCh38.
       - test.sorted.bam.bai: BAM index for 'test.sorted.bam'
+      - NA03697B2_downsampled.pbmm2.repeats.bam: subsample of puretarget pacbio reads from the public pacbio dataset aligned to genome3.fasta
```
Comment (jfy133) [inline README.md:511]: Can you give a link to the public pacbio dataset website/download link and you're good to go :)

**Review by jfy133 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (tanyasarkjain) [reply README.md]: done! :) thanks!

**Pull request merged**